### PR TITLE
1272 - New method for pagination

### PR DESCRIPTION
**Pull request by reshmabidikar**

### CODE REVIEW

File: src/main/java/org/killbill/billing/invoice/api/InvoiceUserApi.java [hunk 1]
```diff
@@ -56,23 +56,32 @@ public interface InvoiceUserApi extends KillbillApi {
...
+    public List<Invoice> getInvoicesByAccount(UUID accountId, boolean includesMigrated, boolean includeVoidedInvoices, boolean includeInvoiceComponents, TenantContext context);
+    
+    /**
+     * @param accountId account id
+     * @param offset the offset of the first result
+     * @param limit the maximum number of results to retrieve
+     * @param context the tenant context
+     * @return paginated invoices
+     */
+    public Pagination<Invoice> getInvoicesByAccountWithPagination(UUID accountId, Long offset, Long limit, TenantContext context);
```
Comment (pierre) [inline src/main/java/org/killbill/billing/invoice/api/InvoiceUserApi.java:72]: @sbrossie Naming is hard... Any thought? What about `getPaginatedInvoicesByAccount`? or `listInvoicesByAccount` (mirrors the `search` APIs)?

Comment (sbrossie) [reply src/main/java/org/killbill/billing/invoice/api/InvoiceUserApi.java:72]: What about:

```
    public Pagination<Invoice> getInvoicesByAccount(UUID accountId, Long offset, Long limit, TenantContext context);
```

(The method signature is different so it works, and we already have multiple `getInvoicesByAccount` so this is consistent with adding one more.

Comment (pierre) [reply src/main/java/org/killbill/billing/invoice/api/InvoiceUserApi.java:72]: 👍

**Comment by reshmabidikar** [comment]
Changed method name to `getInvoicesByAccount`

**Review by pierre — APPROVED** [review]

**Pull request merged**